fiftyone/core/annotation/utils.py

```
@@ -206,3 +206,165 @@ def _is_supported_primitive(field):
             return True
 
     return False
+
+
+# =============================================================================
+# New Attribute Field Helpers
+# =============================================================================
+
+
+def _validate_attribute_entry(attr_name, attr_schema, type_to_ftype):
```

Ruff (0.14.14)

Missing return type annotation for private function `_validate_attribute_entry`

Add return type annotation: `None`

(ANN202)